### ISSUE TITLE
[Manure][EEE] Calculate and report energy costs and use in Manure Module

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#1627]
**Is your feature request related to a problem? Please describe.**
Coordinate with EEE and Manure SMEs to calculate and report energy costs and use associated with separators, treatments, and storages taking into account size, capacity, and frequency of use.

**Describe the solution you'd like**
I'm not sure if this needs to be broken up into smaller issues. 
It seems like a simple solution would be getting the energy costs and use for the different pieces of equipment used for manure separation, treatment, and storage and adding those to input files.

During the simulation the specific separator(s), treatment(s), and storage(s) would be pulled from other input data and checked against the established energy costs/uses list for those types of implements. This could then be reported to OM.

**Comment by Isgotkowitz** [comment]
Grooming session: tagging @KFosterReed, can you please weigh in on the importance and/or feasibility of energy use estimation in here?

**Comment by Isgotkowitz** [comment]
Blocked until there is more SME input about a game plan for estimating energy use in the manure module.

**Comment by KFosterReed** [comment]
I would label this as low priority for right now and until we have an SME with time to dedicate to translating the methods Roohi developed into RuFaS language, I would agree that it is blocked/not feasible at the moment

**Comment by Isgotkowitz** [comment]
Grooming session: blocked by #2092

**Comment by elle-andreen** [comment]
Potentially closed by EEE implementation, but would need to confirm with Jonah or other SME working in EEE!

**Comment by ew3361zh** [comment]
Grooming session:
- @ew3361zh to follow up with @elle-andreen and/or @JoeWaddell about status of this issue.

**Comment by KFosterReed** [comment]
SME Grooming Session: 
- This is still on the ToDo list. Need to check with @JoeWaddell and PML to determine priority level
